examples/shader/shader_material_glsl.rs

```diff
@@ -73,3 +76,16 @@ impl Material for CustomMaterial {
         Ok(())
     }
 }
+
+// This is the struct that will be passed to your shader
+#[derive(AsBindGroup, Clone, TypeUuid)]
+#[uuid = "4ee9c363-1124-4113-890e-199d81b00281"]
+pub struct CustomMaterial {
```

Comment:
Could you move this back above the impl Material?

Reply:
Oh yeah, I realigned with the shader_material example which had it this way.
I'll put it back where it was.

Reply:
Done